### DISCUSSION TITLE
Adding TailwindCSS automatic class sorting

### DISCUSSION THREAD

**Discussion by VictorVSa** [kkinnear/zprint#317] [Ideas]
Hi, I would like to  get some pointers on how to implement a TailwindCSS class sorting, like the plugin for Prettier https://tailwindcss.com/blog/automatic-class-sorting-with-prettier

It's not obvious to me how this could be added to zprint. I have it run automatically after saving, so I thought having it part of the same formatter could be a good idea instead of having it separately. 

Thanks!

**Comment by NoahTheDuke** [comment]
Might be helpful for kkinnear if you add an example of what you have and what you want zprint to produce.

**Reply by kkinnear** [reply]
Thanks!

**Comment by kkinnear** [comment]
I recently added namespace sorting in the `(:require ...)` clause of the `ns` macro, so such things are possible (if difficult).  

That said, I just spent a few minutes looking at the link you included, and I have no idea at all what you would like zprint to do to (I presume) your Clojurescipt code.  As Noah said, above, if you give me an example of your .cljs code today, and what you would like zprint to make it look like tomorrow, I will at least have some idea of what you are asking.  Thanks!
